<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 31, 127);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>550</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(214, 214, 214);
border-top: 10px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>-10</y>
               <width>381</width>
               <height>201</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border-radius: 0px;
	border: 0px solid gray; 
	background-position: left center;
	background-repeat: no-repeat;
	border-top: 0px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="icon_login.qrc">
               <normaloff>:/icon_login/icon/Blue_and_Purple_Astronaut_E-Sports_Team_Logo____5_-removebg-preview.png</normaloff>:/icon_login/icon/Blue_and_Purple_Astronaut_E-Sports_Team_Logo____5_-removebg-preview.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>400</width>
               <height>250</height>
              </size>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="topMargin">
              <number>25</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Seja bem-vindo,</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Identifique-se abaixo para usar a linha de produtos Arcano &lt;/&gt; </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="topMargin">
              <number>15</number>
             </property>
             <property name="rightMargin">
              <number>30</number>
             </property>
             <property name="bottomMargin">
              <number>25</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Usu√°rio</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txt_usuario">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius: 5px;
border: 1px solid gray; 
background-position: left center;
background-repeat: no-repeat;
</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>30</number>
             </property>
             <property name="bottomMargin">
              <number>50</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Senha</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txt_senha">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius: 5px;
border: 1px solid gray; 
background-position: left center;
background-repeat: no-repeat;
</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="styleSheet">
             <string notr="true">border-radius: 5px</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="leftMargin">
              <number>250</number>
             </property>
             <item>
              <widget class="QPushButton" name="btn_login">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton:hover {
	background-color: rgb(170, 0, 127);
 	 border: 1px solid black;
	border-radius: 5px;
}


QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(19,10,41);
	border-radius: 5px;
	border: 1px solid gray; 
	background-position: left center;
	background-repeat: no-repeat;
	border-top: 3px;
}</string>
               </property>
               <property name="text">
                <string>Entrar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icon_login.qrc"/>
 </resources>
 <connections/>
</ui>
